File: server/java/src/main/java/com/onlinepayments/example/service/dtos/additionalpaymentactions/AdditionalPaymentActionResponseDto.java
package com.onlinepayments.example.service.dtos.additionalpaymentactions;

import com.onlinepayments.example.service.domain.common.StatusOutput;
import com.onlinepayments.example.service.domain.common.enums.Status;

public record AdditionalPaymentActionResponseDto(
        String id,
        Status status,
        StatusOutput statusOutput
) { }

File: server/dotnet/sdk-dotnet-example/Business/DTOs/AdditionalPaymentActions/AdditionalPaymentActionResponseDto.cs
using Business.Domain.AdditionalPaymentActions;
using Business.Domain.Enums.AdditionalPaymentActions;

namespace Business.DTOs.AdditionalPaymentActions;

public class AdditionalPaymentActionResponseDto
{
    public string? Id { get; set; }

    public Status? Status { get; set; }

    public StatusOutput? StatusOutput { get; set; }
}